docs/source/tutorial/python/models/basic_mpnn_model.ipynb

```diff
@@ -142,7 +142,7 @@
     }
    ],
    "source": [
-    "MPNN(mp, agg, ffn, batch_norm=False)"
+    "MPNN(mp, agg, ffn, batch_norm=True)"
```

Comment:
I think the original file is okay as `batch_norm` is explicitly set to `False`. I am fine with this change though.

Reply:
The purpose of this change was because in the original notebook, we say "It is on by default but can be turned off" and explicitly turn the `batch_norm` flag to `False`. Since we are inverting it, now the text says "It is off by default but can be turned on.", I think we should explicitly set the flag to `True`; my understanding is that this cell is to demonstrate how to change the `batch_norm` flag.